pypeit/images/sciencecube.py

```diff
@@ -3,7 +3,11 @@
 which is how the ScienceImage is most frequently generated. """
 
 import numpy as np
-from shapely.geometry import Polygon
+
+try:
```

Comment:
Maybe add a note that this is the only usage of this package anywhere in the code base (so far)

Reply:
Also, not sure if you added it to the Install notes.  If not, please do

Reply:
Thanks for the reminder!  Both done.